bugheist/settings.py

```diff
@@ -422,3 +433,12 @@
 STRIPE_LIVE_MODE = False  # Change to True in production
 
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
+
+CALLBACK_URL_FOR_GITHUB = os.environ.get(
+    "CALLBACK_URL_FOR_GITHUB", default="http://127.0.0.1:8000")
```

Comment:
What are these supposed to be?

Reply:
There are the callback urls for github and google, after authentication through google or github, they will redirect a user to some page with a code, we can use that code for access token, also can update that url to any thing where we want user to redirect after authentication.

Reply:
Can you default them to the base url?

Reply:
Yes, we can remail them as it is.

Reply:
Yes we can remail them localhost or default

Reply:
Yes we can remain them localhost or default